firebase-firestore/src/main/java/com/google/firebase/firestore/local/IndexBackfiller.java

```diff
@@ -65,29 +68,6 @@ public void setIndexManager(IndexManager indexManager) {
     this.indexManager = indexManager;
   }
 
-  public static class Results {
-    private final boolean hasRun;
-
-    private final int documentsProcessed;
```

Comment:
FYI I don't think we will ever differentiate between documentsWritten and documentsDeleted as they are handled interchangeably by IndexManager.